_episodes/08-command-substitution.md

```diff
@@ -170,10 +169,10 @@ whitespace to separate command, options (flags) and parameters /
 arguments.  For the same reason it is essential that the command (or
```

Comment:
This section should probably be a callout box rather than a dedicated subsection? GH won't let me comment on the `### **Important**` line above though :(

Reply:
Done.